core/store/src/main/java/uk/gov/gchq/gaffer/store/Context.java

```diff
@@ -34,6 +34,7 @@
  * as the user who executed the operation chain and a map of {@link Exporter}s.
  */
 public class Context {
+    public static final String COMMAND_LINE_ARGS_CONFIG_KEY = "config.command.line.args";
```

Comment:
It might be worth extending this class in the operation-runner module rather than adding this code here.